### PR TITLE
Vendicated.Vencord.Canary version 1.2.9

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107245)

### CODE REVIEW

**Comment by solomoncyj** [comment]
ok

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by solomoncyj** [comment]
https://github.com/microsoft/winget-cli/issues/1885

**Comment by solomoncyj** [comment]
.

**Comment by stephengillie** [comment]
>  Manifest: The filename must match the PackageIdentifier and ManifestType properties from the manifest. actual 'Vencord.Vencord.Canary.locale.en-US.yaml', expected 'Vendicated.Vencord.Canary.locale.en-US.yaml'

File: manifests/v/Vencord/Vencord/Canary/1.2.10/Vencord.Vencord.Canary.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Vendicated.Vencord.Canary
```
Comment (stephengillie) [inline manifests/v/Vencord/Vencord/Canary/1.2.10/Vencord.Vencord.Canary.locale.en-US.yaml:4]: Hi @solomoncyj,

Should these be `Vendicated.Vencord.Canary` or `Vencord.Vencord.Canary`? This is the only location with `Vendicated`.

Comment (lewisakura) [reply manifests/v/Vencord/Vencord/Canary/1.2.10/Vencord.Vencord.Canary.locale.en-US.yaml:4]: Project member here: `Vendicated.Vencord.*` is the correct identifier as we only use `Vencord.*` for organisation projects, whilst Vencord itself (this project) is under Ven's personal user account. The other manifests that have been submitted are incorrect by our standards.

File: manifests/v/Vendicated/Vencord/Canary/1.2.10/Vendicated.Vencord.Canary.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://github.com/Vencord/Installer/releases/download/v1.2.10/VencordInstallerCli.exe
+  InstallerSha256: 2A85002F05EBAA13B0D3F633ABACEC9E9270EA284B97DB1F0AF78E375EFF0B75
+  InstallerSwitches:
+    Silent: -install
+    SilentWithProgress: -install
+    Upgrade: -update
+    Custom: -branch Canary
```
Comment (stephengillie) [inline manifests/v/Vendicated/Vencord/Canary/1.2.10/Vendicated.Vencord.Canary.installer.yaml:18]:
```suggestion
    Custom: -branch canary
```

**Comment by stephengillie** [comment]
Apparently the installer switch branch is case-sensitive:
```
PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Temp\WinGet\Vendicated.Vencord.Canary.1.2.10\VencordInstallerCli.exe -install -branch Canary                                                                                              Using UserConfig                                                                                                        Checking if C:\Users\User\AppData\Roaming\Vencord\dist exists: Yes                                                      Is Dev Install:  false                                                                                                  Fetching https://api.github.com/repos/Vendicated/Vencord/releases/latest                                                Checking if C:\Users\User\AppData\Local\DiscordCanary\app-1.0.75\resources exists: Yes                                  Checking if C:\Users\User\AppData\Local\DiscordCanary\app-1.0.75\resources\app exists: No                               Checking if C:\Users\User\AppData\Local\DiscordCanary\app-1.0.75\resources\app.asar is directory: No                    Found Discord install at  C:\Users\User\AppData\Local\DiscordCanary                                                     The 'branch' flag must be one of the following: [auto|stable|ptb|canary]                                                PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Temp\WinGet\Vendicated.Vencord.Canary.1.2.10\VencordInstallerCli.exe -install -branch canary                                                                                              Using UserConfig                                                                                                        Checking if C:\Users\User\AppData\Roaming\Vencord\dist exists: Yes                                                      Is Dev Install:  false                                                                                                  Fetching https://api.github.com/repos/Vendicated/Vencord/releases/latest                                                Checking if C:\Users\User\AppData\Local\DiscordCanary\app-1.0.75\resources exists: Yes                                  Checking if C:\Users\User\AppData\Local\DiscordCanary\app-1.0.75\resources\app exists: No                               Checking if C:\Users\User\AppData\Local\DiscordCanary\app-1.0.75\resources\app.asar is directory: No                    Found Discord install at  C:\Users\User\AppData\Local\DiscordCanary                                                     Finished fetching GitHub Data                                                                                           Latest hash is c752be4 Local Install is outdated!                                                                       Vencord Installer cli v1.2.10 (89c8345)                                                                                 Patching C:\Users\User\AppData\Local\DiscordCanary...                                                                   Installing latest builds...                                                                                             Downloading file renderer.js.map                                                                                        Downloading file patcher.js.LEGAL.txt                                                                                   Downloading file patcher.js.map                                              
...
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-08-30 16:50:50.878 [CLI ] ShellExecute installer failed: 1 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**